docs/resources/Supported_Devices.md

```diff
@@ -0,0 +1,173 @@
+# Supported Devices {#openvino_docs_OV_UG_supported_plugins_Supported_Devices}
+
+
+@sphinxdirective
+
+
+The OpenVINO runtime can infer various models of different input and output formats. Here, you can find configurations 
+supported by OpenVINO devices, which are CPU, GPU, or GNA (Gaussian neural accelerator coprocessor).
+
+.. note::
+
+   With OpenVINO™ 2023.0 release, support has been cancelled for all VPU accelerators based on Intel® Movidius™.
+
+
++---------------------------------------------------------------------+----------------------------------------------------------------------------------------------------+
+| OpenVINO Device                                                     | Supported Hardware                                                                                 |
++=====================================================================+====================================================================================================+
+|| :doc:`GPU <openvino_docs_OV_UG_supported_plugins_GPU>`             | Intel® Processor Graphics, including Intel® HD Graphics and Intel® Iris® Graphics                  |
```

Comment:
Could you add [ARC](https://www.intel.com/content/www/us/en/products/details/discrete-gpus/arc/arc-a-series/products.html), [FLEX](https://www.intel.com/content/www/us/en/products/details/discrete-gpus/data-center-gpu/flex-series.html) and [MAX](https://www.intel.com/content/www/us/en/products/details/discrete-gpus/data-center-gpu/max-series.html) series to supported HW list?